rapidsms/apps/__init__.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+class RapidsmsConfig(AppConfig):
+
+    name = "rapidsms"
+    default_auto_field = "django.db.models.AutoField"
```

Comment:
Avoids unintended migrations when the default value is eventually changed to `BigAutoField`.